### PR TITLE
[V2API-404]: updated ep 801 to add pagination

### PR DESCRIPTION
**Pull request by nuralogixAGupta**
relates to nuralogix/dfx-api#170

### CODE REVIEW

**Review by anmol-nura — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -2726,15 +2726,21 @@ looking up all studies and current.
...
 + Parameters
+    + Limit (integer, optional) - The number of studies to pull from the list. 
+        + default 25
+    + Offset (integer, optional) - Offest to specify the start of the count.
+        + default 0
     + Status: ACTIVE (string, optional) - If provided, narrows down results to
     templates of a specific StatusID.
     + Type: HEALTH (string, optional) - If provided, narrows down results to
     templates of a specific StudyTypeID.
+    + SortOrder (string, optional) - filter by sorting order (e.g. ASC/DESC).
```
Comment (sambhare) [inline apiary.apib:2742]: Should just say "sorting order ASC or DESC"

File: apiary.apib [hunk 2]
```diff
@@ -2726,15 +2726,21 @@ looking up all studies and current.
...
+    + SortBy (string, optional) - filter by sorting by (e.g. "Name")
```
Comment (sambhare) [inline apiary.apib:2743]: Should say 'key to sort by (e.g. "Name")'

File: apiary.apib [hunk 3]
```diff
@@ -2726,15 +2726,21 @@ looking up all studies and current.
...
 
-### List Templates [GET /studies/templates/{?Status,Type}]
+### List Templates [GET /studies/templates/{?Status,Type,Limit,Offest,SortOrder,SortBy}]
 
 Retrieves a list of study templates that exist in a particular organization.
 
 + Parameters
+    + Limit (integer, optional) - The number of studies to pull from the list. 
+        + default 25
+    + Offset (integer, optional) - Offest to specify the start of the count.
```
[the remark this answers is not in the document]
Comment (nuralogixAGupta) [reply apiary.apib:2736]: fixed

File: apiary.apib (hunk 1 above)
Comment (nuralogixAGupta) [reply apiary.apib:2742]: updated

File: apiary.apib (hunk 2 above)
Comment (nuralogixAGupta) [reply apiary.apib:2743]: updated

File: apiary.apib [hunk 4]
```diff
@@ -2726,15 +2726,21 @@ looking up all studies and current.
             }
         ]
 
-### List Templates [GET /studies/templates/{?Status,Type}]
+### List Templates [GET /studies/templates/{?Status,Type,Limit,Offest,SortOrder,SortBy}]
 
 Retrieves a list of study templates that exist in a particular organization.
 
 + Parameters
+    + Limit (integer, optional) - The number of studies to pull from the list.
```
Comment (daphnezhou) [inline apiary.apib:2734]: The number of study templates to pull from the list.

**Review by anmol-nura — APPROVED** [review]

File: apiary.apib [hunk 5]
```diff
@@ -2726,15 +2726,21 @@ looking up all studies and current.
...
         ]
 
-### List Templates [GET /studies/templates/{?Status,Type}]
+### List Templates [GET /studies/templates/{?Status,Type,Limit,Offset,SortOrder,SortBy}]
 
 Retrieves a list of study templates that exist in a particular organization.
 
 + Parameters
+    + Limit (integer, optional) - The number of studies to pull from the list. 
+        + default 25
```
Comment (RGCsAGupta) [inline apiary.apib:2735]:
```suggestion
        + default returns all the templates
```

**Review by daphnezhou — APPROVED** [review]

**Review by anmol-nura — APPROVED** [review]

**Pull request merged**